src/assets/wise5/directives/wise-tinymce-editor/wise-tinymce-editor.component.ts

```diff
@@ -79,9 +74,6 @@ export class WiseTinymceEditorComponent {
     'wordcount'
   ];
 
-  protected toolbar: string = `undo redo | bold italic underline | numlist bullist`;
-  protected toolbarGroups: any;
-
   constructor(private NotebookService: NotebookService) {
```

Comment:
Lower-case class variable name?
```suggestion
  constructor(private notebookService: NotebookService) {
```